packages/network-controller/tests/provider-api-tests/helpers.ts

```diff
@@ -112,7 +112,7 @@ const mockAllBlockTrackerRequests = ({
   block = DEFAULT_BLOCK,
 }: MockBlockTrackerRequestOptions) => {
   (
-    mockRpcCall({ // eslint-disable-line
```

Comment:
I think this indentation is right.

Reply:
fixed here: https://github.com/MetaMask/core/pull/1064/commits/e3de999197d210517d051b88264277050602057d